### DISCUSSION TITLE
Floating keyboard shows up in mode="time" when clicking the time.

### DISCUSSION THREAD

**Discussion by anieshka** [henninghall/react-native-date-picker#630] [Q&A]
Is there a way to disable this feature? I've recently discovered this feature and didn't notice it before. Can someone help me, please?

**Comment by henninghall** [comment]
Nothing built in, but this might work as a workaround
https://github.com/Alghurairinvestment/react-native-date-picker-zed/commit/de2e5e24eece5fe0c65f9e2d9273e089aab8cdd4
